hw/ip/aon_timer/dv/cov/aon_timer_unr_manually_excluded_excl.el

```diff
@@ -0,0 +1,186 @@
+Condition 4 "937119686" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 3293469597"
+INSTANCE: tb.dut.u_reg.u_wkup_cause_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 5 "937119686" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wkup_ctrl_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wkup_thold_hi_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wkup_thold_lo_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wdog_bark_thold_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wdog_bite_thold_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "1125143680 1408670975"
+INSTANCE: tb.dut.u_reg.u_wdog_ctrl_cdc
+ANNOTATION: "src_ack is only 1 cycle pulse, and is an acknowledgment to a request from a read/write.
+src_busy_q feeds of the request, so by the time src_ack is high src_busy_q should also be high.
+See assertion SrcAckBusyChk_A for more details"
+Condition 3 "86646938" "(src_busy_q && src_ack) 1 -1" (1 "01")
+CHECKSUM: "2815520955 4109606122"
+INSTANCE: tb.dut.u_reg.u_wkup_count_hi_cdc.u_arb
+ANNOTATION: "It can't be hit because dst_update_ack won't be set prior to dst_update_req"
```

Comment:
I'm not sure that's quite correct. These signals come through `prim_sync_reqack`, where `dst_update_req` is visible as `src_req_i` and `dst_update_ack` is visible as `src_ack_o`.

I was slightly surprised that this module doesn't have an assumption that says `src_ack_o -> src_req_i`. But I don't think it's true! If `src_req_i` is asserted and acknowledged, what happens when `src_req_i` drops again?